src/mlgsl_rng.c

```diff
@@ -126,6 +126,7 @@ value ml_gsl_rng_set_default_seed(value seed)
 
 value ml_gsl_rng_alloc(value type)
 {
+  void* x = malloc(2 * 16);
```

Comment:
I was just reading the PR out of curiosity, but I'm puzzled by this line. What is supposed to achieve?